<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>loadFile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoShortOutput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="zoomer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="xMinLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>xMin</string>
    </property>
   </widget>
   <widget class="QLabel" name="xMaxLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>540</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>xMax</string>
    </property>
   </widget>
   <widget class="QLabel" name="yMaxLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>yMax</string>
    </property>
   </widget>
   <widget class="QLabel" name="yMinLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>yMin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>zoom off</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="errors">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
